<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item><widget class="WaveformWidget" name="waveWidget"/></item>
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QComboBox" name="comboWaveType"/></item>
      <item><widget class="QPushButton" name="btnStart"><property name="text"><string>Start</string></property></widget></item>
      <item><widget class="QPushButton" name="btnStop"><property name="text"><string>Stop</string></property></widget></item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QLineEdit" name="editName"/></item>
      <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Save</string></property></widget></item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QComboBox" name="comboSaved"/></item>
      <item><widget class="QPushButton" name="btnLoad"><property name="text"><string>Load</string></property></widget></item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveformWidget</class>
   <extends>QWidget</extends>
   <header>waveformwidget.h</header>
  </customwidget>
 </customwidgets>
</ui>